src/pkg/caendr/caendr/services/sql/etl/homologs.py

```diff
@@ -13,7 +13,7 @@
 C_ELEGANS_PREFIX = 'CELE_'
 C_ELEGANS_HOMOLOG_ID = 6239
 
-TAXON_ID_URL = external_db_url_templates['TAXON_ID_URL']
+TAXON_ID_URL = external_db_url_templates['generic']['TAXON_ID_URL']
```

Comment:
consider renaming upstream the prefix `TAXON_` to `TAXONOMY_` for clarity.

Reply:
I wasn't sure about changing the variable name itself, but I've added comments explaining it's the "Taxonomic ID" URL where relevant.

See: 2b93809e430f7743528be5f0063243675e2a3711.